index.html

```diff
@@ -4,52 +4,119 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ow hey its me</title>
-    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       href="https://fonts.googleapis.com/css2?family=Crimson+Pro:wght@200&family=Outfit:wght@200&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="glitch-text.css" />
+  
   </head>
   <body>
+
+    <!-- header  -->
```

Comment:
if u add more and more content the body of `index.html` will be huge and harder to maintain.
There is ways to include `html template` from different file to an index file.

Divide each part on different file and include them here.

Some of the link:

https://css-tricks.com/the-simplest-ways-to-handle-html-includes/

https://stackoverflow.com/questions/8988855/include-another-html-file-in-a-html-file

Reply:
okay bhaia